activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -270,8 +270,8 @@ def change_column_null(table_name, column_name, null, default = nil) #:nodoc:
 
       def change_column(table_name, column_name, type, **options) #:nodoc:
         alter_table(table_name) do |definition|
-          definition[column_name].instance_eval do
-            self.type = type
+          definition[column_name].instance_eval do |x|
```

Comment:
Do we need the `|x|` here?

Reply:
Whoops, no that was from my debugging, removing now.